setup.py

```diff
@@ -23,17 +23,18 @@
         "numpy>=1.17.2",
         "pandas>=0.25.1",
         "scikit-learn>=0.22.1",
-        "scipy>=1.3.1"
+        "scipy>=1.4.1"
     ],
     extras_require={
         "customplots": [
-            "matplotlib>=3.0.3"
+            "matplotlib>=3.1.3"
```

Comment:
This works when someone installs on py3.5/MacOS?

Reply:
That will have to be set to the lower one, you're right